src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Text } from '../typography/typography';
+
+const Checkbox = (props) => {
+  const [checked, setChecked] = useState(true);
+
+  const handleChange = () => {
+    setChecked(!checked);
+    props.onChange(checked);
+  };
+
+  const HiddenCheckbox = styled.input.attrs({
+    type: 'checkbox',
+    onChange: handleChange,
+  })`
+    height: 1px;
+    width: 1px;
+    margin: -1px;
+    overflow: hidden;
+    position: absolute;
+    visibility: hidden;
+  `;
+
+  const Icon = styled.svg`
+    display: inline-block;
+    fill: none;
+    margin-top: 5px;
+    margin-left: 5px;
```

Comment:
These margins seem to push the checkbox to the bottom and thus break the alignment with the label:
![image](https://user-images.githubusercontent.com/7970458/141799417-76da9b4e-2d69-4a2a-acad-f54b5193a486.png)

Can you finetune this so the checkbox and the label are well aligned?